plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -118,6 +118,9 @@ private void addAppEngineLibrariesToBuildPath(IProject newProject,
       subMonitor.worked(1);
     }
     javaProject.setRawClasspath(newRawClasspath, monitor);
+    
+    Job job = new AppEngineLibraryContainerResolverJob("Initialize libraries", javaProject);
```

Comment:
If the resolution of those artifacts take several seconds/a minute, the user can minimize the progress dialog of this job and start working with the project while it's still in progress.